www/src/pages/en/folder-structure.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Folder Structure
+description: Folder structure of a newly scaffolded Create T3 App
+layout: ../../layouts/docs.astro
+---
+
+The following is the folder structure of a newly scaffolded Create T3 App, with all options selected.
+
+Further down, the description of each folder indicates its purpose and if it is only included with selected libraries.
+
+```
+.
+├─ prisma
+│  └─ schema.prisma
+├─ public
+│  └─ favicon.ico
+└─ src
+   ├─ env
+   │  ├─ client.mjs
+   │  ├─ schema.mjs
+   │  └─ server.mjs
+   ├─ pages
+   │  ├─ _app.tsx
+   │  └─ index.tsx
+   ├─ styles
+   │  └─ globals.css
+   ├─ types
+   │  └─ next-auth.d.ts
+   ├─ utils
+   │  └─ trpc.ts
+   ├─ pages
+   │  └─ api
+   │     ├─ examples.ts
+   │     └─ restricted.ts
+   ├─ server
+   │  ├─ common
+   │  │  └─ get-server-auth-session.ts
+   │  ├─ db
+   │  │  └─ client.ts
+   │  └─ trpc
+   │     ├─ context.ts
+   │     └─ trpc.ts
+   ├─ pages
+   │  └─ api
+   │     ├─ auth
+   │     │  └─ [...nextauth].ts
+   │     └─ trpc
+   │        └─ [trpc].ts
+   └─ server
+      └─ trpc
+         └─ router
+            ├─ _app.ts
+            ├─ auth.ts
+            └─ example.ts
+├─ .env
+├─ .env-example
+├─ .eslintrc.json
+├─ .gitignore
+├─ README.md
+├─ next-env.d.ts
+├─ next.config.mjs
+├─ package.json
+├─ postcss.config.cjs
+├─ prettier.config.cjs
+├─ tailwind.config.cjs
+├─ tsconfig.json
+```
+
+## `prisma`
+
+The `prisma` folder contains the `schema.prisma` file which is used to configure the database connection and the database schema. It is also the location to store migration files and/or seed scripts, if used.
+
+<sub>(With Prisma)</sub>
+
+## `public`
+
+The `public` folder contains static assets that are served by the web server. The `favicon.ico` file is an example of a static asset.
+
+## `src`
+
+The `src` folder contains all the source code of the application.
+
+## `src/env`
+
+Used for environment variable validation and type definitions - see [Environment Variables](usage/env-variables).
+
+### `src/env/client.mjs`
+
+[See the client.mjs documentation](usage/env-variables#servermjs--clientmjs)
+
+### `src/env/server.mjs`
+
+[See the server.mjs documentation](usage/env-variables#servermjs--clientmjs)
+
+### `src/env/schema.mjs`
+
+[See the schema.mjs documentation](usage/env-variables#schemamjs)
```

Comment:
Imo the link to env docs on the H2 is sufficient, no need to link to the individual files?